src/web-auth/captcha.js

```diff
@@ -189,6 +189,23 @@ function injectCaptchaScript(element, opts, callback, setValue) {
       window.arkose = arkose;
       callback(arkose);
     };
+  } else if (opts.provider === AUTH0_V2_CAPTCHA_PROVIDER) {
+    var a0RetryCount = 0;
+    attributes['error-callback'] = function () {
```

Comment:
I don't think this is correct. Are you trying to retry the script loading? That should be done through the attribute 'onerror' like Arkose example above. Also let's try to combine the code since it's identical to Arkose. 

https://developer.mozilla.org/en-US/docs/Web/API/HTMLScriptElement#examples